extensions/opencv/CaptureRTSPFrame.cpp

```diff
@@ -155,12 +172,12 @@ void CaptureRTSPFrame::onTrigger(const std::shared_ptr<core::ProcessContext> &co
   }
 
   frame.release();
-
 }
 
 void CaptureRTSPFrame::notifyStop() {
   // Release the Capture reference and free up resources.
   video_capture_.release();
+  unsetenv("OPENCV_FFMPEG_CAPTURE_OPTIONS");
```

Comment:
The setenv is in initialize(), which is not the logical pair for notifyStop. When the processor is restarted with onSchedule, the env will not be set again. Either we should not unset this, or move the setenv to onSchedule.

Reply:
@nghiaxlee Now that I think of this, we should not unset this at all. If there are two instances of this processor running, and one is stopped through C2, it will unset it for the other processor as well which could cause issues.